base_vat_optional_vies/models/res_partner.py

```diff
@@ -56,3 +62,19 @@ def _construct_constraint_msg(self, country_code):
                 country_code, "'CC##' (CC=Country Code, ##=VAT Number)"
             ),
         )
+
+    @api.model
+    @tools.ormcache("vat")
+    def _check_vies(self, vat):
+        # Check for error of connection
+        try:
+            return super()._check_vies(vat)
+        except Fault as error:
+            if error.args[0] == "MS_MAX_CONCURRENT_REQ":
+                logging.info(
+                    "VIES VAT check not available at this moment as max concurrency "
+                    "requests has been reached."
+                )
+            if error.args[0] == "MS_UNAVAILABLE":
+                logging.info("VIES VAT check not available at this moment.")
+            return False
```

Comment:
I'm so familar with the odoo orm cache but supposed it's better to raise something than cached a falsy value to be able to try it again.

Also generally speaking I prefer something raise than hide it to the user, it will be hard for end user something wrong happens if the user expected the VIE to called and nothing happens on the user interface

Reply:
This method can be called by cron or other functions and a raise will block them.
Moreover, the current function fail quietly in the same way, so if I do that, this method would replicate the same result, making this PR unuseful.

Reply:
The main point here is about using `@tools.ormcache("vat")` that will return `False` does falsy result are cached as well?